CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ storage.
 leading to short session lifetime. Now that this is fixed, the background refresh
 will happen normally, and the session will remain active.
 
+#### node
+
+- The client credential flow as implemented by the CSS Identity Provider is now
```

Comment:
might be good to link to that document describing this?

Reply:
e5795b4f630aecbf8add18a1e2d74e41dd4bf81a